.github/workflows/ci_tests.yml

```
@@ -44,7 +44,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ['3.8', '3.10']
```

```suggestion
        python-version: ['3.8', '3.9', '3.10']
```
I know testing the ends should be fine, but there can always be quirks - so better to test all ranges.